charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,76 @@
+
+# Metadata Schema
+
+## Description
+
+The Human Cell Atlas (HCA) Metadata Schema are JSON format schema. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure [the FAIRness of the HCA data](https://www.nature.com/articles/sdata201618).
+
+These metadata schema aim to capture more than the minimum requirements for representing the samples and experiments conducted in the context of the Human Cell Atlas project but they do not aim to be truely comprehensive and capture every single variable that could be recorded about these samples and experiments. We are aiming for a middle ground that provides data consumers with rich enough information that they can use the data to answer biological questions but to not overburden the data contributors with an expectation to record every detail of every process they run in order to generate this data.
+
+## Objectives
+
+1. Ensuring that HCA data are findable and the schemas are computationally queryable
+2. Ensuring that HCA metadata supports reproducibility and reusability
+3. Ensuring that HCA metadata has a consistent representation and can accurately represent the scientific description of a dataset in a computational readable manner
+4. Defining JSON schema which enable objectives 1 and 2
+5. Defining the semantic rules that the content follows
+6. Working with Ingest Infrastructure to specify the entity structure for HCA metadata schema
```

Comment:
> with Ingest Infrastructure

Maybe change to "with Ingestion Service" to be consistent with how the component is referred to in other charters